src/internal/operations/maintenance_test.go

```diff
@@ -39,8 +40,9 @@ func getKopiaHandles(
 	ctx context.Context, //revive:disable-line:context-as-argument
 ) (*kopia.Wrapper, *kopia.ModelStore) {
 	st := storeTD.NewPrefixedS3Storage(t)
+	repoNameHash := uuid.NewString()[:7]
```

Comment:
`uuid.NewString()[:7]` might be a good candidate to move into a package like `/str/testData` func.  That way all callers can call `strTD.Hash()`, and if we ever need to change the behavior, we only need to change it in the testData func instead of every place that generates a repo name hash.